javalib/src/main/scala/java/lang/System.scala

```diff
@@ -23,7 +23,15 @@ object System {
   def identityHashCode(x: Object): scala.Int =
     x.cast[Word].hashCode
 
-  def getenv(name: String): String                      = ???
+  def getenv(name: String): String = {
+    val cResult = Option(stdlib.getenv(toCString(name)))
+    cResult match {
+      case None    => null
+      case Some(x) => fromCString(x)
+    }
+  }
```

Comment:
1. It should throw a `NullPointerException` if name is null.
2. If you're using `Option`, it's preferable to use `fold` over pattern matching. Otherwise this would also work with simple if-null check.